sml/src/sml_transport.c

```diff
@@ -108,7 +108,7 @@ void sml_transport_listen(int fd, void (*sml_transport_receiver)(unsigned char *
 	unsigned char buffer[MC_SML_BUFFER_LEN];
 	size_t bytes;
 
-	while (1) {
+	while (!feof(fd)) {
```

Comment:
nice try... ;)

suggest to rebase to remove junk from the PR...

Reply:
It will be squashed upon commit anyway, don‘t worty.